Makefile

```diff
@@ -14,4 +15,10 @@ $(CENTOS7_TARGETS): .dapper
 $(CENTOS8_TARGETS): .dapper
 	./.dapper -f Dockerfile.centos8.dapper $(@:centos8-%=%)
 
-.PHONY: $(CENTOS7_TARGETS) $(CENTOS8_TARGETS)
+$(MICROOS_TARGETS): .dapper
+	./.dapper -f Dockerfile.microos.dapper $(@:microos-%=%)
+
+clean:
+	rm -rf dist/ Dockerfile.*.dapper[0-9]*
+
+.PHONY: $(CENTOS7_TARGETS) $(CENTOS8_TARGETS) clean
```

Comment:
Should we be adding the `MICROOS_TARGETS` to `.PHONY`?

Reply:
done